### ISSUE TITLE
A little issue

### ISSUE THREAD

**Issue by BannerGames** [G07cha/MineflayerAutoAuth#1]
When I run the code this appears: bot.chatAddPattern is not a function

**Comment by G07cha** [comment]
Hi @BannerGames, thanks for reporting, could you please provide a minimal reproducible example which causing the issue and also the version of mineflayer that you have? Thanks.

**Comment by BannerGames** [comment]
Hello. A litte example: 
``` 
var bot = mineflayer.createBot({
        username: generateName(),
        host: "ip here",
    })
   require('mineflayer-auto-auth')(bot, 'minecon12');
    bot.on('spawn', function() {
        console.log(bot.username + " entrou no servidor.");
      });
      bot.on('login', () => {
        console.log(bot.username + " entrou pela 1ª vez no servidor.");
      });
     bot.on('serverAuth', function() {
         bot.chat("São apenas testes, não é nada que afete!")
      });
      bot.on('end', () => {
        console.log(bot.username + " saiu do servidor.")  
      });
            bot.on("kicked", (reason) => {
          console.log(bot.username + " foi expulso do servidor.")
          console.log(reason)
      })
```
The version of mineflayer is 2.7.2

**Comment by G07cha** [comment]
Thanks for the response, @BannerGames. I was able to reproduce it locally and it appears like there were some breaking changes which will require some time to fix. A quick solution for you will be to roll back to 1.x.x version of Mineflayer. I'll keep this issue open and in the meanwhile will work on adding support for Mineflayer 2.x.x.

**Comment by G07cha** [comment]
Fixed in 1.0.0, please run `npm install mineflayer-auto-auth@1.0.0` and update your code according to [migration guide](https://github.com/G07cha/MineflayerAutoAuth/blob/master/MIGRATION.md).
